snorkel/contrib/snark/models/candidate.py

```diff
@@ -41,9 +46,9 @@ def __repr__(self):
 # Note: This should ideally not be hard-coded...
 CONTEXT_OFFSET = 2
 # Note: We store the sentence here, not the sentence_id
-SPAN_COLS = ['id', 'sentence', 'char_start', 'char_end', 'meta']
-SENTENCE_COLS = ['id', 'document_id', 'position', 'text', 'words', 
-    'char_offsets', 'lemmas', 'pos_tags', 'ner_tags', 'dep_parents',
+SPAN_COLS = ['id', 'sentence_id', 'char_start', 'char_end', 'meta']
+SENTENCE_COLS = ['id', 'document_id', 'position', 'text', 'words',
```

Comment:
I had to add `abs_char_offsets` here to get the code to run from the start, i.e. not using the CDR backup database. The backup DB does not include this field so presumably this file should be updated. I can do this and add it to this PR, if you like.